content/blog/mlops-–-deploying-an-ml-model-in-greenlake-platform-mlops-service.md

```diff
@@ -0,0 +1,330 @@
+
+**Machine learning Lifecycle**
+
+![ML Life cycle](/img/blog_3.png "ML Life cycle")
+
+The ML project life cycle can generally be divided into three main stages: data preparation, model creation, and deployment. All three of these components are essential for creating quality models that will bring added value to your business. This process is *cyclical* because the insights gained from the existing model will help define the next model to be deployed.
+
+In this article, we focus on optimal model identified after data preparation and building model. More specifically, as an example, we shall deploy a sample ONNX model available in Kubernetes native MinIO Object storage, using Triton Inference Server.
+
+Triton Inference server supports deployment of any AI model from multiple deep learning and machine learning frameworks, including TensorRT, TensorFlow, PyTorch, ONNX, OpenVINO, Python, RAPIDS FIL, and more. Triton delivers optimized performance for many query types, including real time, batched, ensembles and audio/video streaming. To learn more about Triton Inference server, refer to the References section at the end of the post.
+
+HPE GreenLake for MLOps platform allows customers to host their favorite cloud native applications say Mlflow, Minio, etc.
+
+**Pre-requisite**
+
+* An active service subscription to HPE GreenLake for ML Ops
+* MLOps project has been created by MLOps admin and user is able to launch to ECP Portal with MLOps Project Member Role role
+* Access credentials to any S3 based object storage is available
+* Triton Inference Server container image accessible either through on-prem registry (eg Harbor) or public registry accessible.
+
+**Steps to deploy a model**
+
+1)   Validation connection to kubernetes cluster
+
+1. Click HPE GreenLake for ML Ops card in the HPE GreenLake Central **Dashboard** shows the number of previously created projects.
+
+   ![Select MLOps Project](/img/blog_2.png "Select MLOps Project")
+2. Click link “Launch ML Operations Console”
+3. Click “Dashboard” on left navigation of “Ezmeral Container Platform”
+4. Download “kubectl”, “HPE kubectl plugin”, and “kubeconfig”.
+5. Set environment variable KUBECONFIG to point to the kubeconfig file
+6. validate connectivity to the cluster using command “kubectl get no”
+
+2)   Place the model in s3 object storage
+
+1. Place the model and configuration file for triton inference server in object storage
+
+   ![object storage for model](/img/blog_1.png "object storage for model")
+2. Sample configuration file for the model as is below:
+
+\`\``markdown
```

Comment:
Sorry if my previous comment was confusing. Please remove the escape character (the backslash). For a code block, you just need the triple backticks (```markdown) at the beginning of the code snippet and the triple backticks at the end of the code block. 

Make sure to specify appropriate language syntax for the first triple backticks at the beginning of the code block snippet.
Please do the same for all your code blocks  throughout the post.
Thanks,
Denis

Reply:
Yep, this looks good to me.
Thanks!!!